README.md

```diff
@@ -17,6 +17,103 @@ For details on how to contribute, see further down this page.
 ## Developers Reference
 [Reference (available here)](Reference/index.md) is a collection of API references specifically for developers working with and extending Umbraco.
 
+## Extending
+[Extending (available here)](Extending/) is documentation on customizing and extending the backoffice.
+
+## Tutorials
+[Tutorials (available here)](Tutorials/) is a collection of the more extensive tutorials used in the documentation.
+
+# Contributing to the documentation
+Please read our [Contributing Guidelines](CONTRIBUTING.md) to learn how you can get involved and help with the Umbraco Documentation.
+
+# Markdown conventions
+Keep custom HTML to a minimum. All script and style markup are cleaned by default.
+For reference, the [Markdown syntax guide](https://daringfireball.net/projects/markdown/syntax) is available.
+
+## Images
+Images are stored and linked relatively to .md pages, and should by convention always be in an
+`images` folder. So to add an image to `/documentation/reference/partials/renderviewpage.md` you link it like so:
+
+	![My Image Alt Text](images/img.jpg)
+
+And store the image as `/documentation/reference/partials/images/img.jpg`
+
+Images can have a maximum width of **800px**. Please always try to use
+the most efficient compression, `gif`, `png` or `jpg`. No `bmp`, `tiff` or `swf` (Flash), please.
+
+NB: In order to get images to display correctly on GitHub, all image URLs must end with ``.
+
+## External links
```

Comment:
Seems all of the changes here are duplicating the text, think all changes in this file need to be removed again 😉

Reply:
Fixed with 6a1a229